### ISSUE TITLE
Deep Neural Networks - Save and Restore TensorFlow Models

### ISSUE THREAD

**Issue by silverstar085** [udacity/dlnd-issue-reports#166]
In Deep Neural Networks Lesson, under **_Save and Restore TensorFlow Models_**:

In order to save a model, the code says:
```
# Define loss and optimizer
cost = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits, labels))
optimizer = tf.train.GradientDescentOptimizer(learning_rate=learning_rate)\
    .minimize(cost)
```

However, it doesn't work because logits and labels are not defined. It should be:
```
# Define loss and optimizer
cost = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits=logits, labels=labels))
optimizer = tf.train.GradientDescentOptimizer(learning_rate=learning_rate)\
    .minimize(cost)
```

I tried running the code and it works with the change proposed above.

**Comment by clapollo** [comment]
Yep, they changed it for TF 1.0. The original code was written for TF 0.12. I will update this.
